lib/fluent/plugin/in_windows_eventlog.rb

```diff
@@ -92,17 +110,18 @@ def no_encode_record(record)
 
     def start
       super
-      if @pos_file
-        @pf_file = File.open(@pos_file, File::RDWR|File::CREAT|File::BINARY)
-        @pf_file.sync = true
-        @pf = PositionFile.parse(@pf_file)
+      unless @storages.empty?
+        @pf = {}
+        @storages.each {|storage|
```

Comment:
What this code is for? Storage plugins loads data from files at `#start` (via storage plugin helper) in default.